community/values/src/test/java/org/neo4j/values/storable/ValueComparisonTest.java

```diff
@@ -87,7 +84,10 @@
             Float.MAX_VALUE,
             Double.MAX_VALUE,
             Double.POSITIVE_INFINITY,
-            Double.NaN
+            Double.NaN,
+
+            // OTHER
+            null
```

Comment:
Moving NO_VALUE to the end will change behavior from `3.2` in the kernel at least. Does that matter? I don't know. Does cypher 3.2 put nulls last?